web/themes/contrib/civictheme/civictheme_library/components/03-organisms/webform/webform.scss

```diff
@@ -5,6 +5,19 @@
 .ct-webform {
   $root: &;
 
+  .form-checkboxes,
```

Comment:
These are not CivicTheme classes. The challenge needs to take place in the corresponding components.